src/ResourceManager/Websites/Commands.Websites/packages.config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.Storage" version="3.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Websites" version="1.2.0-preview" targetFramework="net45" />
-</packages>
+  <package id="Microsoft.Azure.Management.Websites" version="1.7.1-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@Nking92 same comment about removing the dependencies added in this file since they are defined elsewhere